### PR TITLE
Documentation guideline updated with Installation section

### PR DESCRIPTION
**Pull request by chetkhatri**
What were proposed in this Pull Request ?
* Documentation available at (http://ec2-35-162-199-80.us-west-2.compute.amazonaws.com/static/docs/basic_tutorial.html#) updated with Installation section.
* README.md link for torch-lua updated with installation url.

### CODE REVIEW

**Comment by chetkhatri** [comment]
@alexholdenmiller Could you please take a look on this PR .

File: docs/source/basic_tutorial.rst [hunk 1]
```diff
@@ -16,6 +16,22 @@ Its goal is to provide researchers:
 - multi-task training over many datasets at once
 - seamless integration of Amazon Mechanical Turk for data collection and human evaluation
 
+Install
+-------
+Follow the step by step guide on how to download and install ParlAI.
+
+1. Clone ParlAI Repository:
+```bash
```
Comment (alexholdenmiller) [inline docs/source/basic_tutorial.rst]: needs to be in rst format, same for other lines below:
```
.. code-block:: bash

  git clone https://github.com/facebookresearch/ParlAI.git
```

File: README.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ ParlAI currently requires Python3.
 Dependencies of the core modules are listed in requirement.txt.
 
 Several models included (in parlai/agents) have additional requirements.
-DrQA requires installing [PyTorch](http://pytorch.org/), and the MemNN model requires installing [Lua Torch](http://torch.ch/). See their respective websites for installation instructions.
+DrQA requires installing [PyTorch](http://pytorch.org/), and the MemNN model requires installing [Lua Torch](http://torch.ch/docs/getting-started.html#_). See their respective websites for installation instructions.
```
Comment (alexholdenmiller) [inline README.md]: can you remove the "#_" from the end? same below

File: docs/source/basic_tutorial.rst [hunk 3]
```diff
@@ -16,6 +16,22 @@ Its goal is to provide researchers:
...
+	git clone `<https://github.com/facebookresearch/ParlAI.git>`_
+ ```
+2. Install Required packages for ParlAI:
+```bash pip install -r requirements.txt```
+3. Install ParlAI on Linux:
+```bash python setup.py install```
+4. Several models has additional requirements:
+	a. DrQA requires installing `PyTorch <http://pytorch.org/>`_
```
Comment (ajfisch) [inline docs/source/basic_tutorial.rst]: Might as well add that it also requires spacy for tokenizing. (spacy.io).

**Comment by chetkhatri** [comment]
@alexholdenmiller @ajfisch Updated PR, Kindly review changes made.

**Pull request closed without merging**